Setup/Product.wxs

```diff
@@ -723,8 +723,8 @@
         <Feature Id="ReleaseNotesGenerator" Title="Release notes generator" Level="1">
           <ComponentRef Id="ReleaseNotesGenerator.dll" />
         </Feature>
-        <Feature Id="Stash" Title="Atlassian Stash integration" Level="1">
-          <ComponentRef Id="Stash.dll" />
+        <Feature Id="Bitbucket" Title="Atlassian Bitbucket Server integration" Level="1">
+          <ComponentRef Id="Bitbucket.dll" />
```

Comment:
we need to remove the existing stash.dll as well
there is a section in the definition for removal of old files

Reply:
Did you try generating the installer and see that it removes the old assembly?